src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+
+  sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length)
+    .map((style) => {
+      const [key, value] = style.split(':').map((part) => part.trim());
+
+      cssStyle[key] = value;
```

Comment:
[CODE PRACTICE] - Avoid using loops or methods with side effects for this task. Instead, consider using the reduce method to transform the array of styles into an object without side effects.